src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Repositories/PolicyInformationRepository.cs

```diff
@@ -0,0 +1,226 @@
+using System;
+using System.Threading.Tasks;
+using Altinn.Platform.Authorization.Configuration;
+using Altinn.Platform.Authorization.Repositories.Interface;
+using Altinn.Platform.Storage.Models;
+using Microsoft.Azure.Documents;
+using Microsoft.Azure.Documents.Client;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Altinn.Platform.Authorization.Repositories
+{
+    /// <summary>
+    /// Repository for retrieving instance information
+    /// </summary>
+    public class PolicyInformationRepository : IPolicyInformationRepository
```

Comment:
Why is this callled repository and not storage?

Reply:
It's just a standard naming convention that is used in data layer in the samples, so i reused the same